### ISSUE TITLE
Plan to remove syntax highlighting

### ISSUE THREAD

**Issue by lextm** [vscode-restructuredtext/vscode-restructuredtext#313]
A fix for this has now been implemented and released in upstream. 

@lextm Perhaps it would be simpler to use the [`extensionDependencies` field in extension manifest](https://code.visualstudio.com/api/references/extension-manifest) to let VS Code pull in the syntax highlighting instead? Currently, the two extensions will be competing for the rst files inside VS Code's language resolution engine when both are installed, and people will probably keep posting suggestions like these in downstream. 

This could be achieved by adding 
```json
"extensionDependencies": [
    "trond-snekvik.simple-rst"
]
```
to package.json, and VS Code will automatically install the syntax highlighting extension when the update is pushed.

Up to you though, the current setup also works for me :slightly_smiling_face:

_Originally posted by @trond-snekvik in https://github.com/vscode-restructuredtext/vscode-restructuredtext/issues/309#issuecomment-897138678_

**Comment by lextm** [comment]
Starting from release 154.0.0 this extension uses the same syntax file from [the simple-rst extension](https://github.com/trond-snekvik/vscode-rst).

In the long run, syntax highlighting is going to be removed from here, so that users can install simple-rst separately.

This allows Trond to update the syntax on a different release schedule, and also addresses issues like #210 .

- [x] Test feasibility
- [ ] Wait for https://github.com/trond-snekvik/vscode-rst/issues/7 to be resolved.
- [ ] (optional) Wait for https://github.com/trond-snekvik/vscode-rst/issues/8 to be resolved.
- [ ] Remove syntax highlighting.
- [ ] Add `extensionDependencies`.

**Comment by trond-snekvik** [comment]
I have merged the pull requests in https://github.com/trond-snekvik/simple-rst, and made a couple additional adjustments. The simple-rst extension also provides section symbols, but I've added a check for this extension, and disable the section symbols if both are installed, so users don't get two sets of symbols.

**Comment by lextm** [comment]
Just published 159.0.0 release. It will prompt end users to install simple-rst if not detected.
